colabs/pollinator.ipynb

```diff
@@ -38,16 +38,17 @@
     {
       "cell_type": "code",
       "metadata": {
-        "id": "RcWlME-DwhrS"
+        "id": "RcWlME-DwhrS",
+        "cellView": "form"
       },
       "source": [
         "#@title Config and Requirements\n",
         "\n",
         "!pip install --upgrade pip\n",
-        "!pip install --upgrade typing-extensions==3.10.0.2 protobuf\n",
+        "#!pip install --upgrade typing-extensions==3.10.0.2 protobuf\n",
         "!pip install papermill json5\n",
-        "!pip uninstall -y tqdm\n",
-        "!pip install git+https://github.com/voodoohop/tqdm.git\n",
+        "#!pip uninstall -y tqdm\n",
```

Comment:
@gokaykucuk so basically it does work but it adds a few seconds to the initial startup time of the pollinator. and since we didn't add any frontend code yet that displays this information I commented it out for the time being